extensions/scientific/README.md

```diff
@@ -1,49 +1,51 @@
-# STAC Scientific Extension Spec (`sci`)
+# Scientific Extension Specification (`sci`)
 
 **Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
 
-This document explains the fields of the STAC Scientific Extension to a STAC Catalog. Scientific
-metadata is considered to be data that indicate from which publication a dataset originates and how
-the dataset itself should be cited or referenced. Overall, it helps to increase reproducibility and
-citability.
+Scientific metadata is considered to be data that indicate from which publication a data originates and how
+the data itself should be cited or referenced. Overall, it helps to increase reproducibility and citability.
 
 Human-readable references and [DOIs](https://www.doi.org/) can be used in this extension. DOIs are
 persistent digital interoperable identifier that uniquely identify for digital publications. They
 can be registered at registration agencies affiliated with the
 [International DOI Foundation](https://www.doi.org/).
 
-- [Example](example-merraclim.json)
+- [Example (Collection)](example-merraclim.json)
```

Comment:
Since this example depends on the common properties extension I'd mention that extension somewhere in the intro. And discuss if you have to do things that way, or if this also applies to items, etc. (it's ok to leave this out, since this is just at proposal stage, but I think it'd be good to have)

Reply:
Both are proposals, so I added some information to it. Scientific information are often more closely bound to the collection level anyway, so it should actually be recommended to use commons and collections to specify them there.